docs/src/modules/flavors.md

```diff
@@ -63,6 +63,22 @@ For devices with "boot-as-recovery", the typical LineageOS flashing process invo
 The `boot.img` and `ota` targets can be built using `nix-build ... -A bootImg` or `nix-build ... -A ota`, respectively.
 Check the upstream documentation for your particular device before following the above instructions.
 
+### Note on signing
```

Comment:
So currently, it is a subsection of the lineage os flavor, because I was unsure it applies to every android flavor. My intuition is that it is the case, right? If so, I think this should be moved elsewhere, maybe to `install.md`?